scripts/process.py

```diff
@@ -1,39 +1,60 @@
 import os
+import io
 import csv
 import json
-import xlrd
```

Comment:
@gradedSystem should we remove this from requirements.txt now?

Reply:
@anuveyatsu correct will remove that right now

Reply:
@anuveyatsu fixed